ui.apps/src/main/content/jcr_root/apps/wknd/components/helloworld/helloworld.html

```diff
@@ -13,13 +13,18 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */-->
-<div class="cmp-helloworld" data-cmp-is="helloworld">
-    <h2 class="cmp-helloworld__title">Hello World Component</h2>
+<div class="cmp-helloworld" data-cmp-is="helloworld" data-sly-use.model="com.adobe.aem.guides.wknd.core.models.HelloWorldModel">
+    <h1 class="cmp-helloworld__title">${model.title}</h1>
+    <h3 class="cmp-helloworld__title">${model.subtitle}</h3>
     <div class="cmp-helloworld__item" data-sly-test="${properties.text}">
-        <p class="cmp-helloworld__item-label">Text property:</p>
+        <p class="cmp-helloworld__item-label">Text property 1:</p>
         <pre class="cmp-helloworld__item-output" data-cmp-hook-helloworld="property">${properties.text}</pre>
```

Comment:
Good job 👏 

if you want to learn more about sightly (HTL), sometimes when we know for sure what we're printing / using in the HTL, we can use extra context ...
_For example:_ 
I want to print a number with  ${model.specialinformation} 
We can use ${model.specialinformation @ context='the_right_context'}

[click here to see more about the the_right_context ](https://experienceleague.adobe.com/docs/experience-manager-htl/using/htl/expression-language.html?lang=en#context-settings)